src/components/Body.jsx

```diff
@@ -1,10 +1,20 @@
+import { useState } from "react"
 import ArticlesList from "./ArticlesList"
+import { useEffect } from "react"
+import { getArticles } from "../../utils"
 
 function Body() {
+    const [articles, setArticles] = useState([])
```

Comment:
we should also have a loading state to provide a visual feedback to the user on slow load

Reply:
Added this functionality.